csi-plug-n-play/sidecars/soda-proxy/Readme.md

```diff
@@ -0,0 +1,35 @@
+## soda-proxy
+
+soda-proxy is a simple http server which helps to call the api's of SodaFoundation in a simpler way, it bypasses the authentication and directly helps to connect the osdsapi-server using the client provided by sodafoundation/api.
+
+### Build
+Please follow the below steps to build and run the proxy
+
+```go
+go get github.com/sodafoundation/nbp
+
+cd $GOPATH/src/github.com/sodafoundation/nbp/csi-plug-n-play/sidecars/soda-proxy
+
+go build cmd/proxy.go
+```
+
+Before running soda-proxy you need to export the below variables.(These are the same env variables which are required by osdsctl, for more reference you can see [this](https://github.com/sodafoundation/api/wiki/SODA-Projects-Cluster-Installation-through-Ansible#how-to-test-soda-projects-cluster))
```

Comment:
This wiki page is deprecated and the latest link is https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/
Could you please confirm once with @kumarashit and update

Reply:
Yes @asifdxtreme Please update the link to docs.sodafoundation. The wiki link is deprecated and discouraged to use :)

Reply:
@skdwriting @kumarashit The links have been updated now